mne/preprocessing/ica.py

```diff
@@ -547,6 +547,11 @@ def _fit_epochs(self, epochs, picks, decim, verbose):
         if self.current_fit != 'unfitted':
             self._reset()
 
+        if epochs.events.size == 0:
+            raise RuntimeError('Tried to fit ICA with epochs, but none were '
+                               'found: epochs.events is "{}".'
+                               .format(epochs.events))
+
```

Comment:
Can you add a:
```
with pytest.raises(RuntimeError, match='none were found'):
    ica.fit(epochs)
```
in `mne/preprocessing/tests/test_ica.py`? It's good to add tests even for these failure modes

Reply:
yup, see: https://github.com/mne-tools/mne-python/pull/6660/commits/9c0caf4843354cae68524cbdd43456f5d818056b